lambdas/service/app.py

```diff
@@ -509,7 +509,7 @@ def version():
     }
 
 
-def validate_repository_search(params, **validators):
+def validate_repository_search(entity_type, params, **validators):
```

Comment:
```suggestion
def validate_repository_search(entity_type: EntityType, params, **validators):
```

it's a shame there's no type annotations for the other parameters. Maybe just add `Mapping` for `params`?